portfolio/src/main/webapp/galleryScript.js

```diff
@@ -0,0 +1,20 @@
+// Functions for the slideshows in Gallery
+var slideIndex = 1;
+displaySlide(slideIndex);
+
+function moveSlide(n) {
+  displaySlide(slideIndex += n);
+}
+
+function displaySlide(n) {
```

Comment:
This method feels a bit awkward because it's taking `n` as a parameter, but it decides which slide to show based on `slideIndex` and does not actually update `slideIndex` itself except for in the edge cases.

I'd recommend handling the edge cases in `#moveSlide` and not passing anything into this method.

Reply:
Got it! Changing moveSlide so that it does all of the functionality of changing the slideIndex including the edge cases that were originally in displaySlide. Also making it so that displaySlide takes in no parameters.